### PR TITLE
Cmc param

### PR DESCRIPTION
**Pull request by AshakaE**
# Add optional argument

_Adds optional argument for coimarketcap search_

## How should this be tested?

1. image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/827

### CODE REVIEW

File: src/App/tokenStats/CryptoStats/stats-getter.service.ts [hunk 1]
```diff
@@ -38,8 +35,13 @@ class StatsGetterService {
     return { marketChangeResult, volumeChangeResult }
   }
 
-  async getStats(name: string): Promise<any> {
-    const cmc = await lastValueFrom(this.cmcApiCall(name))
+  async getStats(name: string, cmcName?: string): Promise<any> {
+    let cmc
+
+    if (!cmcName) {
+      cmc = await lastValueFrom(this.cmcApiCall(name))
```
Comment (kesar) [inline src/App/tokenStats/CryptoStats/stats-getter.service.ts:42]: const cmc = (cmcName) ? await lastValueFrom(this.cmcApiCall(cmcName)) : await lastValueFrom(this.cmcApiCall(name))

Comment (AshakaE) [reply src/App/tokenStats/CryptoStats/stats-getter.service.ts:42]: I need this version of linters 😅

File: src/App/tokenStats/CryptoStats/stats-getter.service.ts [hunk 2]
```diff
@@ -13,14 +13,11 @@ class StatsGetterService {
     private configService: ConfigService,
   ) {}
 
-  private cmcApiCall(name: string) {
-    // TEMP HACK
-    const cmcName = (name === 'binancecoin') ? 'bnb' : name;
-
+  private cmcApiCall(name?: string) {
```
Comment (kesar) [inline src/App/tokenStats/CryptoStats/stats-getter.service.ts:16]: this can't be nullable

```suggestion
  private cmcApiCall(name: string) {
```

**Comment by kesar** [comment]
looks good, lets wait till binance demo and then I will merge it 👍🏻

**Pull request merged**